### DISCUSSION TITLE
Different behaviour on different ports

### DISCUSSION THREAD

**Discussion by SchmidtDawid** [tailwindlabs/tailwindcss#2359] [Help]
I played a little with Tailwind and it is so good. I worked on default settings project with vue cli. I make configurations, and on port 8080 everything works fine. My config rules which looks like that:
  ` secondary: 'var(--color-bg-secondary)',`
after serve are rendered as:
```
    .bg-secondary {
      background-color: var(--color-bg-secondary);
    }
```
which is perfect for me. 

But when I change port on another than 8080 by for example:
 `  devServer: { port: 8081 },`

rules are translated on color like:
```
   .bg-secondary {
      background-color: rgb(192, 134, 27);
   }
```

Which is not fine. Why is that hapend?

**Comment by paulpopus** [comment]
Without knowing more about replicating this, could it be browser caching? If you view both pages in incognito, is the value different?

**Reply by SchmidtDawid** [reply]
Its not cache. Effect is the same. 

```
//vue.config.js
module.exports = {
  devServer: {
    port: 8080, //only with this port works properly
    open: true
  },
  lintOnSave: true
};
```


```
//tailwind.config.js
module.exports = {
  purge: [
    './public/**/*.html',
    './src/**/*.vue',
  ],
  theme: {
    extend: {
      textColor:{
        default: {
          '1': 'var(--color-default-1)',
          ...
        },
        primary: 'var(--color-primary)',
        ...
      },
      backgroundColor: {
        primary: 'var(--color-bg-primary)',
        ...
        color: {
          primary: 'var(--color-primary)',
          ..
        }
      },
      borderColor: {
          primary: 'var(--color-border-primary)',
          ...
      }
    },
  },
  variants: {},
  plugins: [],
};
```

and no webpack config different than default generated by vue-cli ("@vue/cli-service": "~4.4.0",)
